.github/workflows/integration-tests.yml

```diff
@@ -43,6 +43,12 @@ jobs:
         run: rm -rf ~/.ansible/test && mkdir -p ~/.ansible/test && ssh-keygen -m PEM -q -t rsa -N '' -f ~/.ansible/test/id_rsa
 
       - name: run tests
-        run: make testall
+        run: |
+          if [ "$RUN_LONG_TESTS" = "1" ]; then
+            make testall
+          else
+            echo "Skipping long tests."
+          fi
```

Comment:
We would still want to run not so long tests in else, right?

Reply:
Good to have if we opt to skip ryt

Reply:
But in the else branch here, it only echo a string?

Reply:
- name: run tests
  run: |
    if [ "$RUN_LONG_TESTS" = "1" ]; then
      make testall
    else
      echo "Skipping long tests."
      make testall
  env:
    LINODE_API_TOKEN: ${{ secrets.DX_LINODE_TOKEN }}
    RUN_LONG_TESTS: ${{ secrets.RUN_LONG_TESTS }}

Reply:
I think in the workflow we can always skip these tests? 

`make testall` always run all tests, I think.
Can we find a way to avoid running the long tests while still going to run the non-long tests?

Reply:
Hi @vshanthe I think we will still want to run some tests even if the long tests are skipped